### DISCUSSION TITLE
MS1 Or MS2 for Abundance Estimation

### DISCUSSION THREAD

**Discussion by DarioS** [vdemichev/DiaNN#148] [General]
The standard approach of deriving the precursor ion quantities involves aggregation of the top 3 fragment ions, which often have very similar average correlation to the fourrth, fifth, sixth fragment ions. Any opinions about using the MS1 Area column instead, thus avoiding choosing a "best fragment" and correlating to it altogether?

**Comment by vdemichev** [comment]
Sometimes using MS2 + MS1 data for quant is indeed beneficial. Ms1.Area alone will typically be more noisy than using MS2, also it's raw non-normalised quantity. There's also an option in DIA-NN --ms1-base-profile which allows to also consider MS1 profile instead of the 'best fragment' profile, but does not seem to be beneficial.
